var/spack/repos/builtin/packages/py-tables/package.py

```diff
@@ -29,8 +29,7 @@ class PyTables(PythonPackage):
     variant('lzo', default=False, description='Support for lzo compression')
 
     # requirements.txt
-    depends_on('python@3.5:', when='@3.4:', type=('build', 'run'))
-    depends_on('python@2.6:', type=('build', 'run'))
+    depends_on('python@2.7:2.7.999,3.0:', when='@3.2.2:', type=('build', 'run'))
```

Comment:
This change doesn't make any sense. Before, you could build older versions with Python 2.6+ and newer versions with Python 3.5+. Now all versions can be built with Python 2.7+, which is not the case.